packages/lisk-transactions/src/errors.ts

```diff
@@ -4,15 +4,21 @@ import { VError } from 'verror';
 export class TransactionError extends VError {
 	public id: string;
 	public dataPath: string;
+	public actual?: string | number | object;
+	public expected?: string | number | object;
 	public constructor(
 		message: string = '',
 		id: string = '',
 		dataPath: string = '',
+		actual?: string | number | object,
+		expected?: string | number | object,
 	) {
 		super(message);
 		this.name = 'TransactionError';
 		this.id = id;
 		this.dataPath = dataPath;
+		this.actual = actual;
+		this.expected = expected;
```

Comment:
Would be nice to implement the .toString function.

Reply:
will be in separate issue #1080